hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenUpdateSpecs.java

```diff
@@ -684,7 +684,8 @@ public HapiSpec updateTokenTreasuryRequiresZeroTokenBalance() {
                         .hasKnownStatus(TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES));
     }
 
-    @HapiTest
+    // To be consistent with the mono behaviour we shouldn't change the memo when we provide empty memo value.
+    // We should think of a method/function to reset the memo value.
```

Comment:
should this test be reenabled and show that the memo is not updated?

Reply:
+1

Reply:
We actually verify that behaviour with the `HapiTest` added in this PR.

Reply:
OK.  In that case can this test be removed?